can-stream-kefir.js

```diff
@@ -82,7 +82,12 @@ canStreamKefir.toCompute = function(makeStream, context){
 	});
 };
 
-/*
- * Exposes a simple toStream method that takes an observable and event or propname and returns a Kefir stream object
- */
-module.exports = canStream(canStreamKefir);
+if (namespace.canStreamKefir) {
```

Comment:
Why can't you have two versions of canStreamKefir? We only `throw` like this for stateful packages that will break apps if multiple versions are loaded.

Reply:
So if you import 2 versions of `can-stream-kefir`, should it overwrite what's already in `can-namespace`?

Reply:
Yes.

Reply:
Upated the PR.